packages/adminrouter/extra/src/nginx.agent.conf

```diff
@@ -58,5 +58,13 @@ http {
             include common/proxy-headers.conf;
             proxy_pass http://metrics/;
         }
+
+        # Group: System
+        # Description: DC/OS version (unauthenticated)
+        location /dcos-metadata/dcos-version.json {
```

Comment:
This would be the first common location.

Looks like Enterprise copies adminrouter, so we'd need to duplicate this PR against it.